internal/util.go

```diff
@@ -40,7 +40,7 @@ type Config struct {
 	MemoryContainer               *fyne.Container
 	SwapBar                       *fyne.Container
 	MemoryBar                     *fyne.Container
-	GPUBar                        *fyne.Container
+	VRAMBar                        *fyne.Container
```

Comment:
The VRAM bar shouldn't need to be added to the app object, since it doesn't need to refresh/have a receiver.

Reply:
make sense. I ll update and squash my commits into 1.